src/index.html

```diff
@@ -1,13 +1,404 @@
-    <title>Miami</title>
+    <title>Bose</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
+
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a href="/" class="logo">
+              <img
+                class="logo__image"
+                src="/images/logo.svg"
+                alt="logo"
+              >
+            </a>
+
+            <a
+            href="tel:+1 234 555-55-55"
+            class="icon icon--call"
+            >
+              <div class="icon__tooltip">
+                +1 234 555-55-55
+              </div>
+            </a>
+
+
+
+            <a
+              href="#menu"
+              class="icon icon--menu"
+            >
+            </a>
+          </div>
+          <h1 class="header__title">
+            The world shades.<br>Your music shines.
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top">
+            <a href="/" class="logo">
+              <img
+                class="logo__image"
+                src="/images/logo.svg"
+                alt="logo"
+              >
+            </a>
+
+            <a href="#close_menu" class="icon icon--cross"></a>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#home" class="menu__link">Home</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#recommended" class="menu__link">Recommended</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#categories" class="menu__link">Categories</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#how-to-buy" class="menu__link">How to buy</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#contact-us" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__call menu__call--call"
+          >
+            +1 234 5555-55-55
+          </a>
+
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__call menu__call--action"
+          >
+            call to order
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section class="page__section recommended" id="recommended">
+        <div class="container">
```

Comment:
This div is unnecessary, you could style the section above instead of adding another wrapper, remove it